### DISCUSSION TITLE
Macan PCM4 GEM Issue

### DISCUSSION THREAD

**Discussion by Quo007** [Mr-MIBonk/M.I.B._More-Incredible-Bash#556] [General]
Hi Everyone, 
My friend and I both own late 2016 Macan with PCM4, I have completed the update on his Macan and he now has Carplay and AA.
Now trying to do my own car, I have completed the updates up to the point where you cancel the backup and it reboots again.
I am unable to access the GEM menu to complete the process, tried the buttons a lot and it will not bring up the green menu.
Having all ready completed this once on my friends Macan I am at a loss as to why I can not enter the GEM menu.
Any help would be very much appreciated.

**Comment by crl0wryjr** [comment]
Bumping old thread 
Same situation, trying to get to GEM screen and it just goes blank. If I hit the scroll wheel again it comes back to the OEM screen.
